### PR TITLE
Problem: Missing Ledger tutorial

### LINKED ISSUE #43: Problem: Missing Ledger tutorial

### PR DESCRIPTION
**Pull request by calvinlauco**
Solution: (Fix #43) Adde Ledger guide with basic operation instructions

### CODE REVIEW

File: docs/wallets/ledger.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# Ledger Hardware Wallets
+
+You can use your [Ledger hardware wallets](https://www.ledger.com/) to securely manage your Testnet CRO assets. Ledger devices are hardware wallets which stores your private keys safely in the device and can be used in even unsafe PCs.
+
+## Pre-requisites
+
+- An initialized Ledger Nano device
+- Install the latest firmware on your device
+- Install Ledger Live on your PC
+- Download our latest [release binary](https://chain.crypto.com/docs/getting-started/croeseid-testnet.html)
```
Comment (lezzokafka) [inline docs/wallets/ledger.md:10]:
```suggestion
- Download our latest [release binary](https://github.com/crypto-com/chain-main/releases)
```

File: docs/wallets/ledger.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+## Install the Ledger app
+
+There are currently two ways to install the Ledger app for Crypto.com Chain:
+
+1. [Ledger Live (Preferred)](#1-install-via-ledger-live)
```
Comment (lezzokafka) [inline docs/wallets/ledger.md:16]:
```suggestion
1. [Ledger Live (Preferred)](#_1-install-via-ledger-live)
```

File: docs/wallets/ledger.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+1. [Build from source code](#2-build-from-source-code)
```
Comment (lezzokafka) [inline docs/wallets/ledger.md:17]:
```suggestion
1. [Build from source code](#_2-build-from-source-code)
```

File: docs/wallets/ledger.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
+# Ledger Hardware Wallets
+
+You can use your [Ledger hardware wallets](https://www.ledger.com/) to securely manage your Testnet CRO assets. Ledger devices are hardware wallets which stores your private keys safely in the device and can be used in even unsafe PCs.
```
Comment (lezzokafka) [inline docs/wallets/ledger.md:3]:
```suggestion
You can use your [Ledger hardware wallets](https://www.ledger.com/) to securely manage your Testnet CRO assets. Ledger devices are hardware wallets which store your private keys safely in the device and can be used in even unsafe PCs.
```

File: docs/wallets/ledger.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+    $ ./chain-maind keys add [name] --coin-type=[coin-type] --account=[account-index] --ledger --keyring-backend=test
+    ```
+
+    - `name` is an arbitrary name for your new key
+    - `coin-type` is the Coin type. Depends on the application you installed, please use different types:
+        | App | Coin type |
+        | --- | --- |
+        | Cosmos| `118` |
+        | Crypto.com Chain | `394` |
+    - `account-index` can be any integer inclusively starting from `0` to `2147483648`
```
Comment (lezzokafka) [inline docs/wallets/ledger.md:56]: Can add sth like: 

`keyring-backend` is the keyring's storage backend, we will use `--keyring-backend test` here for simplicity.

File: docs/wallets/ledger.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+    ```bash
+    $ ./chain-maind keys add Ledger --coin-type=118 --account=0 --ledger --keyring-backend=test
+    ```
+1. An address will be displayed on your Ledger device
+1. Confirm the address on your Ledger device and verify the address displayed on the terminal. They should match with each other.
+1. Your address is generated successfully. You can now use the address for any chain operations.
+
+## Query account balance
+
+After you have [generated an address](#generate-address), you can query its balance by running in terminal:
```
Comment (lezzokafka) [inline docs/wallets/ledger.md:67]:
```suggestion
After you have [generated an address](#generate-address), kindly use the [CRO faucet](https://chain.crypto.com/faucet) to obtain test CRO tokens. Once the transaction is completed, you can query its balance by running in terminal:
```

File: docs/wallets/ledger.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+```bash
+$ ./chain-maind query bank balances [address]
+```
+
+`address` is the address you have generated. If you didn't copy your address, you can query the address by running in terminal:
+
+```bash
+$ ./chain-maind keys list
```
Comment (lezzokafka) [inline docs/wallets/ledger.md:76]:
```suggestion
$ ./chain-maind keys list --keyring-backend=test
```

**Review by leejw51 — APPROVED** [review]
lgtm

**Review by lezzokafka — APPROVED** [review]

**Pull request merged**